code/classes/DMSDocumentCart.php

```diff
@@ -21,11 +21,23 @@ public function __construct($backend = null)
     /**
      * Returns all the cart items as an array
      *
-     * @return array
+     * @return ArrayList
      */
     public function getItems()
     {
-        return $this->backend->getItems();
+        $result = $this->backend->getItems();
+
+        return $result;
+    }
+
+    /**
+     * Returns all the cart items as an ArrayList
+     *
+     * @return ArrayList
+     */
+    public function getItemsAsArrayList()
+    {
+        return ArrayList::create($this->getItems());
```

Comment:
I don't think we need two methods returning almost the same thing - if you need an array list at some point in the code just do it there `$arrayList = ArrayList::create($this->getItems())` instead of adding an extra method